pydicom/valuerep.py

```diff
@@ -606,13 +604,38 @@ def _encode_personname(components, encodings):
 
 
 class PersonName:
-    def __new__(cls, *args, **kwargs):
+    """Representation of the value for an element with VR **PN**."""
+    # Is this supposed to be mutable or immutable?
```

Comment:
Good question - I would say immutable makes more sense.

Reply:
I think immutable - basically a string with extra behavio(u)rs.

Reply:
OK, I'll switch the `original_string` attribute over to a property getter.

The same question could be asked about IS/DS and the date/time classes, should I do the same there with `original_string`?

Reply:
Or maybe it should just be private? Its a weird attribute.

Reply:
I'll leave it alone for now